commands/cluster_commands/create

```diff
@@ -239,10 +239,10 @@ civo_cluster_create() {
 
   if ! eval "$(getGumSpinnerOrLogger "checking if grpl is installed") kubectl  get configuration.pkg.crossplane.io grpl 2>/dev/null"; then
     if [ "${AUTO_CONFIRM}" == "true" ]; then
-      grpl cluster install --TARGET_PLATFORM="CIVO" --CIVO_CLUSTER="${CLUSTER_NAME}" --CIVO_REGION="${CIVO_REGION}" --CIVO_API_KEY="${CIVO_API_KEY}" --AUTO_CONFIRM="true" --CIVO_EMAIL_ADDRESS="info@grapple-solutions.com"
+      grpl cluster install --TARGET_PLATFORM="CIVO" --CIVO_CLUSTER="${CLUSTER_NAME}" --CIVO_REGION="${CIVO_REGION}" --CIVO_API_KEY="${CIVO_API_KEY}" --AUTO_CONFIRM="true" --CIVO_EMAIL_ADDRESS="info@grapple-solutions.com" --INSTALL_KUBEBLOCKS="true"
```

Comment:
@zahid-iqbal-alpha 
why do we use the "info@grapple-solutions.com" here?
I think in this case we need to ask the user for his identification.

in the case of civo, is it possible to automatically retrieve the email address of the civo account?
if no email could be retrieved, then we should ask for the email address in the process.

Reply:
sure, we will use customer's email here.